sdk/src/main/java/com/kin/ecosystem/marketplace/view/MarketplaceFragment.java

```diff
@@ -43,9 +44,9 @@ public static MarketplaceFragment newInstance(INavigator navigator) {
 	private INavigator navigator;
 
 
-	private TextView spendSubTitle;
+//	private TextView spendSubTitle;
```

Comment:
Please remove all comments, if you will want to revert it then you can always get it from the history